src/components/Icon/Identity.tsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable react/no-unknown-property */
+import React from 'react'
+
+function Identity() {
+  return (
+    <svg width="418" height="185" viewBox="0 0 418 185" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is this svg this image?
<img width="628" alt="image" src="https://github.com/decentraland/governance/assets/8242162/4cd18f51-d210-4f33-92ac-47279e1a4752">
It's not an icon 🤔